src/NuGet.Clients/VisualStudioAPI/IPackageManagerProvider.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System.Runtime.InteropServices;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace NuGet.VisualStudio
+{
+    /// <summary>
+    /// Interface for integrating other package manager
```

Comment:
Provide a longer description here, comments on public VS interfaces will become the documentation.